### ISSUE TITLE
Reprojection Image Seems Strange

### ISSUE THREAD

**Issue by monamourvert** [acfr/cam_lidar_calibration#44]
Hi, I applied your code to my own data an strangely I got this result. I worked with your noetic branch with Azure Kinect Camera and Ouster Lidar.

My reprojection result:
![image](https://github.com/acfr/cam_lidar_calibration/assets/121480926/71fd0a32-d338-43f2-a26f-d1c3de81e2e2)

Since I'm new at this, I just wondering is this kind of result is normal? I tried several times, and most of them produce similar result. The only different is the place of the yellow and the green cross sometimes not lies on the center of board. Is there anyone also experience this strange result?

**Comment by monamourvert** [comment]
Another thing that I found, no matter perfectly I isolate the chessboard, the dimension error is quite large. 
So, to summarize it:
1. I calculate the offset of the chessboard as accurate as I can for the params.yaml
2. The camera_info is directly retrieved from /rgb/camera_info from Azure Kinect so I assume there is nothing wrong with the intrinsic parameter.
3. I tried to isolate the chessboard as detail as I can. This one is just an example (I screenshot this as fast as I can)
![image](https://github.com/acfr/cam_lidar_calibration/assets/121480926/34a4ecce-b74e-4bcb-94df-5a7477792f3c)
![image](https://github.com/acfr/cam_lidar_calibration/assets/121480926/076ffc89-2943-42b2-9f6d-6e8ef67252fb)
But I guess the dot that mark the board area seems having a big translation from the edge. I don't I'm not so sure.
4. The dimension error of the board is strangely always > 40
![image](https://github.com/acfr/cam_lidar_calibration/assets/121480926/392d1d0e-1526-4351-a576-a17d3c7d1523)

Hope somebody can help me about this. Thanks!
